### ISSUE TITLE
Overheating.

### ISSUE THREAD

**Issue by ifiplease** [kyujin-cho/pixel-volte-patch#218]
I use a pixel 7, after following all the steps, it didn't work and I guess my carrier doesn't support it. But also, after I did it, I noticed the temperature of my phone gets very high so that my phone refuses to charge unless I turn it off.
So my question is, is there a way to reverse everything that was done? Thanks.

**Comment by fillwithjoy1** [comment]
Yes, go to the app > Select your carrier from the bottom > Miscellaneous (scroll to the bottom) > Reset all settings
